src/components/coursCard.tsx

```diff
@@ -0,0 +1,57 @@
+import FlagEN from "@/assets/flag_en.svg";
+import Image from "next/image";
+import { FaRegStar } from "react-icons/fa";
+import { v4 as uuidv4 } from 'uuid';
+
+
+export const CoursCard:FC<{data: coursDetails}> = ({data})=>{
+    return <div className="w-[400px] h-[300px] border-black rounded-md border-[3px] flex flex-col">
+        {/* card title */}
+        <div className="bg-[#FFDF36] p-3 flex justify-between flex-row-reverse items-center border-black border-[3px] rounded-[4px] border-t-0 border-r-0 border-l-0 basis-1/6">
+            {data.title}
+            { data.level === "سهل" && <p className="rounded-xl border border-black py-1 px-2 bg-[#A2FF93]">{data.level}</p>}
+            { data.level === "متوسط" && <p className="rounded-xl border border-black py-1 px-2 bg-[#FFEE93]">{data.level}</p>}
+            { data.level === "صعب" && <p className="rounded-xl border border-black py-1 px-2 bg-[#FF9393]">{data.level}</p>}
+        </div>
+        {/* card body */}
+        <div className="text-right flex flex-col justify-between basis-2/3 p-3">
+            <p>{data.description}</p>
+            <div>
+                <p className="font-bold text-[30px]">{data.rating}</p>
+                <div className="flex justify-between items-center flex-row-reverse">
+                    <div className="flex flex-row-reverse items-center gap-2">
+                        <div className="flex flex-row-reverse">
+                            {Array.from(Array(5)).map((_, i)=>{
+                                if(i+1 <= data.rating)
+                                    return <FaStar color="#FFDF36" className="w-[30px] h-[30px]"/>;
+                                else
+                                    return <FaRegStar color="#FFDF36" className="w-[30px] h-[30px]"/>;
+                            })}
+                        </div>
+                        <p>({data.nbrOfRaters} تقيم)</p>
+                    </div>
+                    <div className="flex justify-center items-center gap-3">
+                        {data.languages.map((lang)=>{
+                            if(lang === "arabic")
+                                return <Image src={FlagAR} alt="flag" className="w-[30px] h-[30px]"/>
+                            else if(lang === "english")
+                                return <Image src={FlagEN} alt="flag" className="w-[30px] h-[30px]"/>
+                        })}
+                    </div>
+                </div>
+            </div>
+            <div className="flex flex-row-reverse">
+                <div className="flex -space-x-1 overflow-hidden">
+                    {Array.from(Array(4)).map(()=><img className="inline-block h-6 w-6 rounded-full ring-2 ring-white" src="https://images.unsplash.com/photo-1491528323818-fdd1faba62cc?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80" alt="" key={uuidv4()}/>)}
+                </div>
+                <p>٢٠٠+ ملتحق</p>
```

Comment:
take the number from your demo data

Reply:
i guess the the `nbrOfRaters` should be string instead of number I don't know if the we can automatically transform arabic number to latin? if so, I can keep the the same type for `nbrOfRaters`

Reply:
yes you can 
try this
![image](https://github.com/Warriors-of-Light/sahwa/assets/64937092/00e2ecef-3daf-4e06-99ed-4c95e7cac33a)

or you can iterate over the number as a string and change each char using a mapping object = 
{
'1': ١,
'2': ٢,
...
}